.github/workflows/iroha2-dev-pr-title-commit-msg.yml

```diff
@@ -92,3 +92,67 @@ jobs:
       - name: None of the above
         if: steps.fix-match.outputs.match == '' && steps.refactor-match.outputs.match == '' && steps.feature-match.outputs.match == '' && steps.docs-match.outputs.match == '' && steps.ci-match.outputs.match == ''
         run: exit 1
+
+  commit-message-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.after }}
+
+      - name: Get the last commit message
+        uses: mathiasvr/command-output@v1
+        id: commit-msg
+        with:
+          run: git log -1 --pretty=format:"%s" ${{ github.event.after }}
+
+      - name: Feature
+        id: feature-match
+        uses: actions-ecosystem/action-regex-match@v2.0.2
+        with:
+          text: ${{ steps.commit-msg.outputs.stdout }}
+          regex: '^\[feature\] #\d+(, #\d+)*: .+$'
+
+      - name: Refactor
+        id: refactor-match
+        if: steps.feature-match.outputs.match == ''
+        uses: actions-ecosystem/action-regex-match@v2.0.2
+        with:
+          text: ${{ steps.commit-msg.outputs.stdout }}
+          regex: '^\[refactor\]( #\d+(, #\d+)*)?: .+$'
+
+      - name: Fix
+        id: fix-match
+        if: steps.refactor-match.outputs.match == ''
+        uses: actions-ecosystem/action-regex-match@v2.0.2
+        with:
+          text: ${{ steps.commit-msg.outputs.stdout }}'
+          regex: '^\[fix\] #\d+(, #\d+)*: .+$'
+
+      - name: Documentation
+        id: docs-match
+        if: steps.feature-match.outputs.match == ''
+        uses: actions-ecosystem/action-regex-match@v2.0.2
+        with:
+          text: ${{ steps.commit-msg.outputs.stdout }}
+          regex: '^\[documentation\]( #\d+(, #\d+)*)?: .+$'
+
+      - name: CI
+        id: ci-match
+        if: steps.feature-match.outputs.match == ''
+        uses: actions-ecosystem/action-regex-match@v2.0.2
+        with:
+          text: ${{ steps.commit-msg.outputs.stdout }}
+          regex: '^\[ci\]( #\d+(, #\d+)*)?: .+$'
+
+      - name: Merge
+        id: merge-match
+        if: steps.feature-match.outputs.match == ''
+        uses: actions-ecosystem/action-regex-match@v2.0.2
+        with:
+          text: ${{ steps.commit-msg.outputs.stdout }}
+          regex: '^Merge .+$'
```

Comment:
Remove. We shouldn't have merge commits at all.

Reply:
So, does it mean that nobody should update his PR branch wit the target branch through Merge commit?
Like `Merge remote-tracking branch 'upstream/iroha2-dev' into feature...`

Reply:
Yes, our Git workflow has changed and updates by __merge__ are now deprecated. Instead, it is recommended to __rebase__ each update @BAStos525

Reply:
> Yes, our Git workflow has changed and updates by merge are now deprecated.

Small correction. Hyperledger never allowed merge commits to begin with. The merge commits have been deprecated since Iroha 1, but not disabled.